ui/redux/selectors/claims.js

```diff
@@ -731,6 +731,16 @@ export const selectPreorderTagForUri = createCachedSelector(selectMetadataForUri
   if (matchingTag) return matchingTag.slice(9);
 })((state, uri) => String(uri));
 
+export const selectProtectedContentTagForUri = createSelector(
+  selectMetadataForUri,
+  (metadata: ?GenericMetadata) => metadata && new Set(metadata.tags).has(MEMBERS_ONLY_CONTENT_TAG)
+);
+
+export const selectedRestrictedCommentsChatTagForUri = createSelector(
+  selectMetadataForUri,
+  (metadata: ?GenericMetadata) => metadata && new Set(metadata.tags).has(RESTRICTED_CHAT_COMMENTS_TAG)
+);
+
```

Comment:
:small_blue_diamond: Minor;  what's the benefit of the extra conversion to `Set` for this case, vs. just `includes`?
- I think we don't care about duplicates here (just looking if one exists), and
- The conversion probably negatef any performance improvements from `has` for this specific case.